README.md

```diff
@@ -53,6 +53,8 @@ Explore questions and conversations on our [Discourse forum](https://jaas.ai/doc
 
 Developers are welcome to read through our [contributing guidelines](CONTRIBUTING.md) to learn how to make code changes.
 
+File bugs at our project [Launchpad](https://bugs.launchpad.net/juju/+bugs) site
```

Comment:
Could you please deep link to https://bugs.launchpad.net/juju/+filebug on that line? The Launchpad UI can be pretty bewildering if you haven't seen it before. 

```markdown
[File bugs](https://bugs.launchpad.net/juju/+filebug) at our [Launchpad project](https://bugs.launchpad.net/juju/+bugs) 
```

Reply:
Done